include/nccl_ofi.h

```diff
@@ -60,6 +60,9 @@ extern "C" {
 /* Maximum number of grouped receives */
 #define NCCL_OFI_MAX_RECVS	1
 
+/* Maximum length of inline write in bytes */
+#define NCCL_OFI_MAX_INLINE_WRITE_BYTES 8
```

Comment:
I'm probably missing some background here, but I don't understand what this variable does. I don't see it used at all in the plugin, which makes me wonder whether it belongs in the plugin.

Reply:
It's going to be used to implement inline_write for neuron in a subsequent commit.